STYLE_API_TEMPLATE.md

```diff
@@ -27,10 +27,11 @@ The following sections describe the basic API documentation structure. Each sect
 1. Name of API (heading level 2)
 2. Path parameters (heading level 3)
 3. Query parameters (heading level 3)
-4. Sample request (heading level 4)
-5. Sample response (heading level 4)
-6. Request body (heading level 3)
-7. Sample request (heading level 4)
+4. Request body (heading level 3)
+5. Sample request (heading level 4)
+6. Sample response (heading level 4)
+7. Response body (heading level 3)
+8. Sample response (heading level 4)
```

Comment:
Remove this line - sample response is mentioned twice

Reply:
Removed line "8".